packages/data-mate/src/function-configs/date/getTimeBetween.ts

```diff
@@ -0,0 +1,78 @@
+import { FieldType, TimeBetweenFormats } from '@terascope/types';
+import { getTimeBetween } from '@terascope/utils';
+import {
+    ProcessMode, FunctionDefinitionType, FunctionDefinitionCategory, FieldTransformConfig
+} from '../interfaces';
+
+export interface GetTimeBetweenArgs {
+    start?: Date | number | string;
+    end?: Date | number | string;
+    readonly format: TimeBetweenFormats;
+}
+
+export const getTimeBetweenConfig: FieldTransformConfig = {
```

Comment:
This will fix the type error you are getting:

```suggestion
export const getTimeBetweenConfig: FieldTransformConfig< GetTimeBetweenArgs > = {
```